content/eventslist/2019-10-03-whatsinatestautomationframework.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "What's In A Test Automation Framework?"
+date: "2019-09-25T15:21:34.217Z"
+startDate: "2019-10-03T15:20:01.000Z"
+startTime: "6:00pm"
+endDate: "2019-10-03T15:20:01.000Z"
+endTime: "8:00pm"
+locationName: "Glassdoor"
+locationStreet: "13"
```

Comment:
sigh... I accidentally hit 'enter'
locationStreet should be:
1330 W Fulton St Floor 6, Chicago, IL 60607